.vscode/tasks.json

```diff
@@ -0,0 +1,25 @@
+{
+    // See https://go.microsoft.com/fwlink/?LinkId=733558
+    // for the documentation about the tasks.json format
```

Comment:
No such thing as comments in JSON :(

Reply:
There is in visual studio json.

Reply:
> Looks good. Is this work in progress or do you want to merge?
> 
> Just asking because I'm wondering about documentation, and if it lives in this PR or a future one. For example I've seen a gitpod "badge" they suggest for sticking in READMEs; there might be something to say about the build command (even screenshots?).

So, to test the .gitlab.yaml, it needs to pre-exist in a branch when you launch, which is why I had to push it.

But also, I was playing with gitpod/vscodes default PR workflow. I did click draft I think, but it seems not to have applied it.
 
> My other question is if the gitpod stuff is / can be cached, or does it mean people have to wait a minute before all the docker pulling is finished, first time they run it? If so, another thing to mention in docs when we do them.

So, yes, if we set up some further things, but I'm just trying to figure out how it all fits together atm.

Reply:
Very much trying things out atm, don't even know if what I've added works atm

Reply:
> Very much trying things out atm, don't even know if what I've added works atm

Spoiler: it didn't :)

Reply:
> Very much trying things out atm, don't even know if what I've added works atm
Oh sorry I mistook this for a PR requiring a review.